### PR TITLE
Player movement

### PR DESCRIPTION
**Pull request by n2oneProgrammer**

### CODE REVIEW

File: models/objects/player.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        move_collision_handler.begin = self.block_move_start
+        move_collision_handler.separate = self.block_move_end
+
+        ball_collision_handler = space.add_collision_handler(ObjectCollisionType.PLAYER, ObjectCollisionType.BALL)
+        ball_collision_handler.begin = self.player_dead
+
+    def run_events(self, event):
+        if event.type == pygame.KEYDOWN:
+            if event.key == pygame.K_LEFT:
+                self.velocity = Vector2(-300, 0)
```
Comment (jaanonim) [inline models/objects/player.py]: create move_speed variable for easy access

**Review by jaanonim — CHANGES_REQUESTED** [review]

**Review by jaanonim — APPROVED** [review]
LGTM

**Review by mwawrzen — APPROVED** [review]
LGTM

**Review by Adron4444 — APPROVED** [review]
LGTM

**Pull request merged**